CHANGELOG.md

```diff
@@ -3,10 +3,23 @@
 NOTE: From this version on the frameworks changed to Dynamic instead of Static library
 
 ### Bug fixes
-- 
+- In implicit authentication flow, completion block of the original request will only get notified once with cancellation of authentication flow, or actual result of the endpoint. Previously, the original request got notified in every error that happened during the authentication flow. [DE311261]
+- Logout, and login notification are now being broadcasted correctly. Previously, the notifications were being sent at incorrect time, so SDK was representing incorrect status upon these notifications. [DE310488]
```

Comment:
Logout and login notifications now broadcast correctly. Previously, notifications were not sent at the correct time, and the SDK also represented incorrect status for these activities.